### ISSUE TITLE
How to get 'umap' env

### ISSUE THREAD

**Issue by MichaelGMoore** [sg-s/umap-matlab-wrapper#12]
I am just a humble stupid man. I do not have a 'umap' env in conda. How to get this?

(base) mmoore@main-workstation:/opt$ conda env list
# conda environments:
#
base                  *  /opt/anaconda3

(base) mmoore@main-workstation:/opt$ conda.getenv

**Comment by sg-s** [comment]
you have to make a new environment, see here:

https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment

**Comment by MichaelGMoore** [comment]
Holy crud it worked.  Thanks!  I would have known this if Python guy, but I'm Matlab guy.

Putting steps here in case any other poor fools have same silly question:

Steps:
(after install srinivas.gs_mtools, condalab, umap-matlab-wrapper with "git clone" and add them to matlab path)
in terminal:
(base) > conda create -n umap python
(base) > conda activate umap
(umap) > pip install umap-learn

Then in matlab get:
> conda activate umap
> u = umap()

u = 

  umap with properties:

             n_neighbors: 15
            n_components: 2
                  metric: 'euclidean'
           learning_rate: 1
                min_dist: 0.1000
                  spread: 1
        set_op_mix_ratio: 1
      local_connectivity: 1
      repulsion_strength: 1
    negative_sample_rate: 5
    transform_queue_size: 4
      target_n_neighbors: -1
           target_weight: 0.5000
          transform_seed: 42
                  labels: [0x1 double]
          containing_dir: '/opt/umap-matlab-wrapper'
